### PR TITLE
B3 Reports Cleanup

### PR DESCRIPTION
**Pull request by biyeun**
- removes Bootstrap 2 from reports
- unsplits b2/b3 files
- `is_bootstrap3` no longer necessary for reports. Bootstrap 3 by default

plan is to put this on staging, poke around. Since all the reports are b3 at this point, it should be just a matter of double checking to make sure no typos were made.

best reviewed commit-by-commmit. everything should be nicely siloed

@orangejenny 
cc @proteusvacuum

### CODE REVIEW

**Comment by biyeun** [comment]
AW YEAH 🎉 
screen shot 2016-05-03 at 4 33 40 pm

**Comment by dannyroberts** [comment]
This is awesome! 💃

File: custom/care_pathways/templates/care_pathways/partials/report_table.html [hunk 1]
```diff
@@ -21,10 +21,10 @@
             <tr>
                 {% for header in report_table.headers %}
                     <th {% if not report_table.pagination.is_on %}data-sort="{{ header.sort_type }}" data-sortdir="{{ header.sort_direction }}"{% endif %} {% if header.css_class %}class="{{ header.css_class }}"{% endif %}>
-                        <i class="icon-white"></i>
+                        <i class="icon-white fa dt-sort-icon"></i>
```
Comment (orangejenny) [inline custom/care_pathways/templates/care_pathways/partials/report_table.html]: isn't `icon-white` no longer necessary?

File: custom/world_vision/templates/world_vision/multi_report.html [hunk 2]
```diff
@@ -81,7 +81,7 @@ <h3 class="media-heading">{{ title }}<br/></h3>
                 {% endif %}
 
                 {% if report_table.bad_request_error_text %}
-                    badRequestErrorText: "<span class='label label-important'>Sorry!</span> {{ report_table.bad_request_error_text }}",
+                    badRequestErrorText: "<span class='label label-danger'>Sorry!</span> {{ report_table.bad_request_error_text }}",
```
Comment (orangejenny) [inline custom/world_vision/templates/world_vision/multi_report.html]: trans?

**Comment by orangejenny** [comment]
This is awesome! Really impressed by the scope of this whole migration, and by how methodical it's been.

**Comment by orangejenny** [comment]
Added no-merge label for now so it can be tested on staging. But the code is good to go from my perspective.

**Comment by orangejenny** [comment]
Couple of things I found grepping around in `corehq/apps/reports/templates`
- Can https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/reportdata/data_export.html be deleted?
- Should https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/async/print_report.html still be including bootstrap2_js.html?
- Should https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/form/partials/single_form_tree.html be migrated? It has a couple of `span6` elements.

File: custom/world_vision/templates/world_vision/multi_report.html (hunk 2 above)
Comment (biyeun) [inline custom/world_vision/templates/world_vision/multi_report.html]: It's a custom report, so I just left that alone

**Comment by biyeun** [comment]
> Can https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/reportdata/data_export.html be deleted?

I think that view is still being used for permissions. I wanted to leave that out of this particular PR, as I wanted to do this quickly and get it in.

> Should https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/async/print_report.html still be including bootstrap2_js.html?

Also left out to address on a different PR

> Should https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/form/partials/single_form_tree.html be migrated?

I'll look at this one.

**Comment by biyeun** [comment]
> Should https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/form/partials/single_form_tree.html be migrated?

This is actually being used in cloudcare, so i'm going to look at it the spans separately.

File: custom/care_pathways/templates/care_pathways/partials/report_table.html (hunk 1 above)
Comment (biyeun) [inline custom/care_pathways/templates/care_pathways/partials/report_table.html]: this will be a separate cleanup to make sure the sorting works without icon-white in there. The dt-sort-icon is a placeholder but I wanted to do it in a separate PR as it spans across more than just reports.

**Comment by biyeun** [comment]
@orangejenny tested a few of each kind of report on staging to make sure they load with no js errors. worked splendidly. I think this is g2g.

**Comment by proteusvacuum** [comment]
:tada:
